### PR TITLE
[libgnutls] Update to 3.8.7.1

### PR DESCRIPTION
**Pull request by dg0yt**

### CODE REVIEW

**Comment by talregev** [comment]
There is a libgnutls 3.8.7:
https://mirrors.dotsrc.org/gcrypt/gnutls/v3.8/

**Comment by dg0yt** [comment]
It wasn't there when I started ;-)

**Comment by talregev** [comment]
> It wasn't there when I started ;-)

Yes, I understand. It release today.

File: ports/libgnutls/portfile.cmake [hunk 1]
```diff
@@ -5,15 +5,15 @@ vcpkg_download_distfile(tarball
...
+    SHA512 672d4085d950dbe4aecb105b398458745a1e5cec67b4171a7916daf87762f21db275f677fe048fb8323c52e201ea3da92efd02d20e4cae19a1fe6535723b2bc4
 )
 vcpkg_extract_source_archive(SOURCE_PATH
     ARCHIVE "${tarball}"
     SOURCE_BASE "v${VERSION}"
     PATCHES
         ccasflags.patch
         use-gmp-pkgconfig.patch
-        link-zlib.patch   # directly as before 3.8.4
+        compression-libs.diff
```
Comment (dg0yt) [inline ports/libgnutls/portfile.cmake:16]: This simple patch should be enough.
I found https://gitlab.com/gnutls/gnutls/-/merge_requests/1867 now. So in the next release, the patch can be removed.

**Comment by dg0yt** [comment]
> There is a libgnutls 3.8.7: https://mirrors.dotsrc.org/gcrypt/gnutls/v3.8/

And now 3.8.7.1 is rolling in ...

**Comment by talregev** [comment]
> > There is a libgnutls 3.8.7: https://mirrors.dotsrc.org/gcrypt/gnutls/v3.8/
> 
> And now 3.8.7.1 is rolling in ...

I understand you. Thank you for your effort 🙏🏻

**Comment by talregev** [comment]
> > There is a libgnutls 3.8.7: https://mirrors.dotsrc.org/gcrypt/gnutls/v3.8/
> 
> And now 3.8.7.1 is rolling in ...

I don't see the 3.8.7.1 yet.
search on download link, GitHub and GitLab source code

**Review by LilyWangLL — APPROVED** [review]

**Review by Hoaholy — COMMENTED** [review]
- /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"

**Pull request merged**